src/open_company/api/companies.clj

```diff
@@ -96,26 +100,75 @@
   :delete! (fn [_] (company/delete-company slug))
 
   ;; Create or update a company
+  ;; TODO remove possibility to create company
```

Comment:
That's why it's "in theory" and "if we allow". I'm happy to see the PUT go for create. Real question I think is should PUT stay for update, and if so, then we can't really prevent it as an attempted create, since it's there. We'd have to either make it work, or at least have to deal with it as a case that shouldn't work.